dev/benchmarks/complex_layout/pubspec.yaml

```diff
@@ -16,16 +16,18 @@ dependencies:
   #   flutter update-packages --force-upgrade
   flutter_gallery_assets: 1.0.2
 
-  archive: 3.3.2 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
+  archive: 3.3.4 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
```

Comment:
It looks like archive added pointycastle as a dependency. Until we decide what to do here, I would recommend pinning the version and filing a bug

Reply:
Agreed; I was planning to investigate this tomorrow.